src/Speckle.Objects/Structural/CSI/Geometry/CSINode.cs

```diff
@@ -40,7 +41,14 @@ public CSINode(
       ?? new Axis(
         "Global",
         AxisType.Cartesian,
-        new Plane(new Point(0, 0), new Vector(0, 0, 1), new Vector(1, 0, 0), new Vector(0, 1, 0))
+        new Plane
+        {
+          origin = new Point(0, 0),
+          normal = new Vector(0, 0, 1),
+          xdir = new Vector(1, 0, 0),
+          ydir = new Vector(0, 1, 0),
+          units = Units.Meters, //Not sure if defaulting to meters is correct, but it was what we were doing previously inside Plane's ctor
+        }
```

Comment:
It is most likely not. TBH I'm leaning towards nuking all structural objects (at least the app specific ones) to be able to start anew and not have to drag these refactors into classes we're most likely not keeping anyway.

Reply:
Lets chat sometime, 
we'll need to nuke some eventually anyway, but I'm a little hesitant to start lighting the dynamite just yet.